src/components/Home/BannerStandard.tsx

```diff
@@ -0,0 +1,41 @@
+interface BannerInfo {
```

Comment:
Add this component into routes so we can see its effect
Also you may want to pull from dev first so some of the styling changed can be updated